src/Dependencies/Composition/project.json

```diff
@@ -0,0 +1,9 @@
+{
+  "supports": {},
+  "dependencies": {
+    "Microsoft.Composition": "1.0.27"
```

Comment:
We've already run into situations where versions get updated in master, but projects in future get missed. Now, you update in once place and when it moves to future all places get updated.